resources/data/contributors.json

```diff
@@ -1145,6 +1145,23 @@
           -26.62229
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "Name": "Nicolas Godet",
+        "Committer": "No",
+        "First Commit Message": "Fix qgis#41977",
```

Comment:
I realise it's not the exact commit message, but I'd rather see something more descriptive here like "Fix QgsCellStatisticsAlgorithm::name() should not return a translatable string".